tests/network/vmi_networking.go

```diff
@@ -408,7 +408,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 	Context("VirtualMachineInstance with dhcp options", func() {
 		It("[test_id:1778]should offer extra dhcp options to pod iface", func() {
 			libnet.SkipWhenClusterNotSupportIpv4()
-			dhcpVMI := libvmifact.NewFedora(libnet.WithMasqueradeNetworking(), libvmi.WithResourceMemory("1024M"))
```

Comment:
nit: The commit message says "The change also modifies some minor newline occurrences."

Is it something that was dropped in the rebase?

Reply:
@EdDev Hi, can you please update the commit message?

Reply:
Done